app/views/comments/create.js.erb

```diff
@@ -4,5 +4,4 @@
   $('#comment-<%= @comment.reply_to %>-reply').before('<%= escape_javascript(render :partial => "notes/comment", :locals => {comment: @comment, answer_id: @answer_id}) %>')
 <% end %>
 $('#answer-<%= @answer_id %>-comment-section .help-block').remove();
-$('textarea').val("");
```

Comment:
and likewise here with something like `#comment-<%= @comment.reply_to %>-reply`?

Reply:
Do you think you could try it out on your local copy of the app? Have you been able to install it? Thanks a ton!

Reply:
Yes, I will try it this weekend. Thank you for your help

Reply:
Hi, just checking if you've gotten stuck on this at all, or if I could help in any way? Thanks!